### PR TITLE
feat: add spec and parser for /etc/audisp/audispd.conf

### PR DESCRIPTION
**Pull request by JoySnow**
### All Pull Requests:

Check all that apply:

* [x] Have you followed the guidelines in our Contributing document, including the instructions about commit messages?
* [ ] Is this PR to correct an issue?
* [ ] Is this PR an enhancement?

### Complete Description of Additions/Changes:

Add new spec and parser for file `/etc/audisp/audispd.conf`

### CODE REVIEW

**Comment by JoySnow** [comment]
Hi @xiangce , could you please help review this PR?

File: insights/parsers/auditd_conf.py [hunk 1]
```diff
@@ -1,11 +1,25 @@
...
+Audit Conf files parsers
+========================
 
 The auditd.conf file is a standard key = value file with hash comments.
 Active settings are provided using the `get_active_settings_value` method or
 by using the dictionary `contains` functionality.
 
+The audispd.conf file has the same format and usage with auditd.conf.
+
+Note:
```
Comment (xiangce) [inline insights/parsers/auditd_conf.py:11]: `.. note::` would be betrer

File: insights/parsers/auditd_conf.py [hunk 2]
```diff
@@ -1,11 +1,25 @@
...
+    For Red Hat Enterprise Linux 7 and older, auditd and audispd are separate
+    processes.
+    Starting with Red Hat Enterprise Linux 8 the functionality of audispd has
+    been migrated to auditd.
+
+AuditdConf - file ``/etc/audit/auditd.conf``
+-------------------------------------------------
```
Comment (xiangce) [inline insights/parsers/auditd_conf.py:18]: length 😃

File: insights/parsers/auditd_conf.py [hunk 3]
```diff
@@ -1,11 +1,25 @@
...
+
+AudispdConf - file ``/etc/audisp/audispd.conf``
+-------------------------------------------------
```
Comment (xiangce) [inline insights/parsers/auditd_conf.py:21]: and length again

File: insights/parsers/auditd_conf.py (hunk 1 above)
Comment (JoySnow) [reply insights/parsers/auditd_conf.py]: Thanks @xiangce , updated in newest commit.

File: insights/parsers/auditd_conf.py (hunk 2 above)
Comment (JoySnow) [reply insights/parsers/auditd_conf.py]: Forget of this checking, corrected in newest commit.

File: insights/parsers/auditd_conf.py (hunk 3 above)
Comment (JoySnow) [reply insights/parsers/auditd_conf.py]: Corrected in newest commit.

**Comment by JoySnow** [comment]
Hi @xiangce , Updated based on your suggestions. Could you help review the changes?

**Review by xiangce — APPROVED** [review]

**Comment by xiangce** [comment]
test me

**Comment by xiangce** [comment]
Only RHEL610 failed in IQE test due to env issue, ignore it.

**Comment by xiangce** [comment]
test me

**Pull request merged**